packages/@aws-cdk-testing/cli-integ/tests/cli-integ-tests/proxy.integtest.ts

```diff
@@ -0,0 +1,125 @@
+import { promises as fs } from 'fs';
+import * as path from 'path';
+import { integTest } from '../../lib/integ-test';
+import { startProxyServer } from '../../lib/proxy';
+import { TestFixture, withDefaultFixture } from '../../lib/with-cdk-app';
+
+const docker = process.env.CDK_DOCKER ?? 'docker';
```

Comment:
I suppose the answer is yes, but does this work in an ECS container? Do we need any special setup?